docs/faq/nuclei.md

```diff
@@ -58,17 +58,17 @@
 
 	!!! check "HTTP Traffic"
 
-		Out of the box, nuclei makes around **1200 - 1300** HTTP requests in total for a given target when ==all== available public templates are executed, which is considered relatively low in numbers and will grow as the template number grows.
+		Nuclei usally makes fewer HTTP requests than the number of templates selected for a scan due to its intelligent request reduction. While some templates contain multiple requests, this rule generally holds true across most scan configurations.
 
 	!!! check "Safe Templates"
 
-		Nuclei templates project houses a variety of templates which includes fuzzing and templates resulting DOS on the target system. To ensure that no one accidentally runs these templates, we tagged and excluded these templates from the default nuclei scan. These templates can be only executed when the user explicitly instructs nuclei to run them.
+		The nuclei templates project houses a variety of templates which include fuzzing and templates which may result in a DOS against the target system. To ensure that no one accidentally runs these templates, they are tagged and nuclei excludes them from the default nuclei scan. These templates can be only executed when the user explicitly instructs nuclei to run them using the `itag` option.
```

Comment:
"_`nuclei` excludes them from the default `nuclei` scan_" is a bit too repetitive